docs/rbac.yaml

```diff
@@ -0,0 +1,63 @@
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: kube-aws-iam-controller
+  namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kube-aws-iam-controller
+rules:
+- apiGroups:
+  - "amazonaws.com"
```

Comment:
Can we really claim this group name? I think it's best left for software actually published by the trademark holder. Perhaps something involving "kube," "iam," and "aws" would be safer.

Reply:
Good point, I also thought about this but went with it anyway since kube2iam does the same and no complaints from AWS :) But you are right, let's use a less trademark infringing value.

Reply:
`mikkel.labs/aws-iam`?